airflow/jobs/scheduler_job.py

```diff
@@ -1348,11 +1315,11 @@ def query(result, items):
                 self._change_state_for_executable_task_instances(items,
```

Comment:
This variable name is not valid because this method now returns TaskInstance rather than SimpleTaskInstance.

Reply:
https://github.com/apache/airflow/pull/6792/files#diff-c35269bcfbbe386e269ffa7487e86192R1238